packages/suite/src/actions/wallet/send/sendFormActions.ts

```diff
@@ -135,18 +140,25 @@ export const handleAddressChange = (outputId: number, address: string) => (
     dispatch: Dispatch,
     getState: GetState,
 ) => {
-    const { account } = getState().wallet.selectedAccount;
-    if (!account) return null;
+    const { send, selectedAccount } = getState().wallet;
+    if (!send || selectedAccount.status !== 'loaded') return null;
+    const { account } = selectedAccount;
+    const { networkType } = account;
+    const output = getOutput(send.outputs, outputId);
 
     dispatch({
         type: SEND.HANDLE_ADDRESS_CHANGE,
         outputId,
         address,
         symbol: account.symbol,
-        networkType: account.networkType,
+        networkType,
         currentAccountAddress: account.descriptor,
     });
 
+    if (networkType === 'ripple') {
+        dispatch(debounce(rippleActions.checkAccountReserve(output.id, address), 700));
```

Comment:
told ya it will not work like that! 😁 I changed the interval to 1500 just to be sure and it keeps firing on every key stroke. check out the usage of debounce in next wallet

Reply:
i like the idea of checking only valid addresses

Reply:
I don't check the account if there is an error. Soooo I just remove the debounce.